files/en-us/web/http/headers/want-repr-digest/index.html

```diff
@@ -0,0 +1,25 @@
+---
+title: Want-Repr-Digest
+slug: Web/HTTP/Headers/Want-Repr-Digest
+page-type: http-header
+browser-compat: http.headers.Want-Repr-Digest
+spec-urls: https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-12
+status:
+  - experimental
+---
+
+{{HTTPSidebar}}{{SeeCompatTable}}
+
+The **`Want-Repr-Digest`** request or response header states the wish for a {{HTTPHeader("Repr-Digest")}} header.
```

Comment:
Is this really a response header (as well)? (asking, not stating). I should check spec, but I have other things on my queue today.

```suggestion
The **`Want-Content-Digest`** request header indicates a desire for the response to include a {{HTTPHeader("Content-Digest")}} header.
```

Reply:
Yes; I am currently also not quite sure what its use would be:
> Senders can indicate their interest in Integrity fields and hashing
> algorithm preferences using the Want-Content-Digest or Want-Repr-
> Digest fields. These can be used in **both requests and responses**.

Cf. [draft-12, 4. Integrity preference fields](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-12#name-integrity-preference-fields) (emphasis mine)

Reply:
Thanks for checking. I've emailed the spec authors.